src/features/automaticTheaterMode/index.ts

```
@@ -3,6 +3,7 @@ import type { YouTubePlayerDiv } from "@/src/types";
 import { isLivePage, isWatchPage, waitForSpecificMessage } from "@/src/utils/utilities";
 
 export async function enableAutomaticTheaterMode() {
+	if (!isWatchPage()) return;
```

ESLint

[error] Insert `␍`

(prettier/prettier)